packages/react/src/components/Loading/Loading-test.js

```diff
@@ -7,11 +7,51 @@
 
 import React from 'react';
 import Loading from '../Loading';
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
+describe('Loading - accessibility', () => {
+  describe('Automated Accessibility Testing', () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(<Loading id="test-id" />);
+      document.body.appendChild(wrapper.getDOMNode());
+    });
+
+    it('should have no Axe violations', async () => {
+      await expect(document).toHaveNoAxeViolations();
+    });
+
+    it('should have no DAP violations', async () => {
+      await expect(document.getElementById('test-id')).toHaveNoDAPViolations(
+        'Loading'
+      );
+    });
+  });
+
+  describe('Screenreader Accessibility', () => {
+    let wrapper;
+    let getLoader;
+
+    beforeEach(() => {
+      wrapper = shallow(<Loading id="test-id" />);
+      getLoader = () => wrapper.find('#test-id');
+    });
+
+    it('has a programatically determinable label', () => {
```

Comment:
Should we tie these to specific requirements or guidelines? Maybe links to wherever we are defining these or where we are getting them from?